src/AppBundle/Controller/ArticleInfoController.php

```diff
@@ -76,9 +77,14 @@ public function resultAction(Request $request)
         $project = $request->attributes->get('project');
         $page = $request->attributes->get('article');
 
-        // sets the database name within $this->lh
-        $dbValues = $this->lh->databasePrepare($project, 'ArticleInfo');
-        $projectUrl = $dbValues['url'];
+        $projectData = ProjectRepository::getProject($project, $this->container);
+
+        if (!$projectData->exists()) {
+            $this->addFlash("notice", ["invalid-project", $project]);
+            return $this->redirectToRoute("articleInfo");
```

Comment:
I don't think we should redirect in GET requests like this, as it breaks navigation unnecessarily. Instead, if a project doesn't exist, the user should be shown the same form with all their values, as well as an error message, so they can more easily fix whatever typo they made.